### PR TITLE
apiextensions.k8s.io/v1 and --crd-version flag added

### PR DESCRIPTION
**Pull request by sheriff-rh**
Docs JIRA: 
https://issues.redhat.com/browse/OSDOCS-1065

SDK JIRA: 
https://issues.redhat.com/browse/OSDK-987

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@joelanford please take a look when you get a chance, thanks! This is in regards to https://issues.redhat.com/browse/OSDK-987.

**Comment by sheriff-rh** [comment]
> @joelanford please take a look when you get a chance, thanks! This is in regards to https://issues.redhat.com/browse/OSDK-987.

Hi Joe! Would you mind having a look? Thanks!

**Review by codyhoag — COMMENTED** [review]
One suggestion for avoiding parentheses in running text. Everything else LGTM!

File: modules/osdk-cli-reference-new.adoc [hunk 1]
```diff
@@ -22,6 +22,9 @@ The `operator-sdk new` command creates a new Operator application and generates
 |`--api-version`
```
Comment (joelanford) [inline modules/osdk-cli-reference-new.adoc:22]: The `new` subcommand no longer exists in v0.19.x. We should remove docs about it.

File: modules/osdk-cli-reference-new.adoc [hunk 2]
```diff
@@ -22,6 +22,9 @@ The `operator-sdk new` command creates a new Operator application and generates
...
 |CRD `APIVersion` in the format `$GROUP_NAME/$VERSION`, for example `app.example.com/v1alpha1`. Used with `ansible` or `helm` types.
 
+|`--crd-version`
+|CRD version to generate (e.g., `v1`). Default setting is `v1beta1`.
+
 |`--generate-playbook`
 |Generate an Ansible playbook skeleton. Used with `ansible` type.
```
Comment (joelanford) [inline modules/osdk-cli-reference-new.adoc:30]: In v0.19.x, we removed support in the `new` subcommand for Go projects. In line 53, we should remove the references to `--type=go`

Comment (sheriff-rh) [reply modules/osdk-cli-reference-new.adoc:30]: Per our discussion on Slack, I'll leave the `new` subcommand in place for Ansible and Helm projects at this time. Thanks for reviewing, Joe!

**Pull request merged**